hasjob/views/campaign.py

```diff
@@ -180,7 +180,7 @@ def action_new(self, **kwargs):
     @route('views.csv')
     def view_counts(self, **kwargs):
         campaign = self.obj
-        timezone = current_auth.actor.timezone if current_auth else 'UTC'
+        timezone = current_auth.actor.timezone if current_auth.is_authenticated else 'UTC'
```

Comment:
Was this in the original code? I suggest removing `current_auth` entirely until we address this.

Reply:
@jace yeah, this was in the original code. there are 5-6 places where current_auth is used even before I opened that PR, as in they are deployed to production and been working. I didn't touch those.